qt/aqt/stats.py

```diff
@@ -154,6 +153,9 @@ def __init__(self, mw: aqt.main.AnkiQt) -> None:
         self.name = "deckStats"
         self.period = 0
         self.form = aqt.forms.stats.Ui_Dialog()
+        # Hack: Switch out web views dynamically to avoid maintaining multiple
+        # Qt forms for different versions of the stats dialog.
+        self.form.web = LegacyStatsWebView(self.mw)
```

Comment:
Constructing a web view twice is not ideal, but I couldn't think of a better solution without splitting Qt forms for the new and legacy stats dialogs in some way.